benchmarks-mobile/protolite_app/app/src/main/java/io/grpc/grpcbenchmarks/ProtobufBenchmarker.java

```diff
@@ -1 +1,223 @@
+package io.grpc.grpcbenchmarks;
 
+import com.google.protobuf.CodedInputStream;
+import com.google.protobuf.CodedOutputStream;
+import com.google.protobuf.MessageLite;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.util.zip.GZIPInputStream;
+import java.util.zip.GZIPOutputStream;
+
+/**
+ * Created by davidcao on 6/13/16.
+ */
+public class ProtobufBenchmarker {
+    private static final long MIN_SAMPLE_TIME_MS = 2 * 1000;
+    private static final long TARGET_TIME_MS = 10 * 1000;
+
+    public static BenchmarkResult serializeProtobufToByteArray(final MessageLite msg)
+            throws Exception {
+        final MessageLite clonedMessage = msg
```

Comment:
Seems like I accidentally left this in from when we were trying to see if we could somehow bypass protolite caching message size. I'll remove this